account/user.go

```diff
@@ -165,3 +165,18 @@ var TestUser = User{
 	Email:    "testdeveloper@testalm.io",
 	Identity: TestIdentity,
 }
+
+// TestIdentity2 only creates in memory obj for testing purposes
+var TestIdentity2 = Identity{
```

Comment:
I just wanted to raise the hand :). I guess we should try to tackle this in a further iteration.

I saw that there are defined some Test variables in `user.go`. I believe those variables should be defined inside the test Go files like the `user_test.go`. This is a weird practice but I saw you did it because you wanted to export them in other Tests.

This problem is due to package name differs from the one of the package of the other Go files, e.g. `user.go` --> package `account` while `user_test.go` --> package `account_test`. The majority of the repos follows an approach that uses the same package name but I saw few with this different package name. A solution could be to import the test package `import github.com/almighty/almighty-core/account_test` or rename the test files to use the same package name of the package where they live.